README.md

```diff
@@ -42,16 +42,68 @@ On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that resid
 - `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
 - `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
 
+#### How to enable/disable kubemacpool for a namespace 
+
+Kubemacpool is disabled by default on a new namespace.
+To enable kubemacpool on a specific namespace:
+```bash
+kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocateForAll mutatevirtualmachines.kubemacpool.io=allocateForAll
+namespace/example-namespace labeled
+```
+
+To disable kubemacpool in a namespace:
+```bash
+kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
+namespace/example-namespace labeled
+```
+
+**note:** If a VMI is created directly and not through a VM, then it is handled in kubemacpool by the pod handler.
+
+#### How to change the opt-in label value on kubemacpool
```

Comment:
This does not answer how to make this into opt-out, but we can add it in a different PR. No need to block this one.

Reply:
I think it's there in the note. no?

Reply:
> if the kubemacpool's mutatingwebhookconfiguration `kubemacpool-mutator` namespace-selector value per vm/pod is set to `allocateForAll`, then you can also opt-out your namespace by setting the label value to `disable` in your namespace

The value is set to `allocateForAll` by default, so what should I change?